package.json

```diff
@@ -32,16 +32,17 @@
     "events": "1.1.1",
     "homoglyph-finder": "1.1.1",
     "identicon.js": "github:status-im/identicon.js",
-    "instabug-reactnative": "2.8.1",
+    "instabug-reactnative": "2.12.0",
     "level-filesystem": "1.2.0",
+    "metro": "^0.30.2",
```

Comment:
@MaxRis how was metro coming in before if it is just being added to package.json now? 🤔

Reply:
@MaxRis, I also have a question for package versions. In `develop` branch we have:
```
"react": "^16.2.0",
"react-dom": "16.2.0",
```
Should we update them? Seems that they are successful with react-native 0.55.4
Asking because I'm merging develop into desktop now and want to understand what versions to leave.

Reply:
@rcullito `metro` package was added automatically after `npm install` command run. As prerequisite, I have deleted package-lock.js file before running `npm install`. Previously, metro was installed as dependency of react-native-desktop module. Not sure why `npm install` generated the record explicitly now.

Reply:
ah yes, alright that makes complete sense wrt it being a dependency. thanks for clarifying @MaxRis !

Reply:
@vkjr if to specify version as `^16.2.0` recommended version of npm (5.5.1) complains that react-native module has unmet dependency of react@16.3.1 and this results at least into issue that `metro` package installed inside `node_modules/react-native/node_modules/metro` directory, but not inside 'node_modules/metro'.
Above issue also caused hanging of Jenkins desktop builds.
Probably, you can keep react version from this PR and experiment with others versions once everything else will work fine, hopefully